src/readme.html

```diff
@@ -3,96 +3,96 @@
 <head>
 	<meta name="viewport" content="width=device-width" />
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-	<title>WordPress &#8250; ReadMe</title>
-	<link rel="stylesheet" href="wp-admin/css/install.css?ver=20100228" type="text/css" />
+	<title>ClassicPress &#8250; ReadMe</title>
+	<link rel="stylesheet" href="wp-admin/css/install.css?ver=20181217" type="text/css" />
 </head>
 <body>
 <h1 id="logo">
-	<a href="https://wordpress.org/"><img alt="WordPress" src="wp-admin/images/wordpress-logo.png" /></a>
+	<a href="https://www.classicpress.net/"><img alt="ClassicPress" src="wp-admin/images/cp-logo-gradient.svg" /></a>
 </h1>
-<p style="text-align: center">Semantic Personal Publishing Platform</p>
+<p style="text-align: center">The business-focused CMS.<br>Powerful. Versatile. Predictable.</p>
 
-<h2>First Things First</h2>
-<p>Welcome. WordPress is a very special project to me. Every developer and contributor adds something unique to the mix, and together we create something beautiful that I&#8217;m proud to be a part of. Thousands of hours have gone into WordPress, and we&#8217;re dedicated to making it better every day. Thank you for making it part of your world.</p>
-<p style="text-align: right">&#8212; Matt Mullenweg</p>
+<h2>A word from our Founder</h2>
+<p>[we need a paragraph intro text from Scott for this]</p>
+<p style="text-align: right">&#8212; Scott Bowler</p>
```

Comment:
It's probably going to be best to wait for a separate PR for a word from the founder.

Reply:
Yes, exactly, so that is why I already filled in the blanks, bit silly to keep Matt's name there, no?

Reply:
so leaving this for Scott then, he can send something that we can then add as a separate PR.

Reply:
@senlin whatever we get done in this PR is probably going to ship in v1, so it should not be "we need a paragraph from Scott for this".  Can you work with @scottybo to get something we can drop in here, or should we remove this section?